builder/internals.go

```diff
@@ -489,8 +489,11 @@ func (b *Builder) processImageFrom(img *imagepkg.Image) error {
 		b.Config = img.Config
 	}
 
-	if len(b.Config.Env) == 0 {
-		b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
+	// No default path on Windows (set by HCS)
+	if runtime.GOOS != "windows" {
+		if len(b.Config.Env) == 0 {
+			b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
```

Comment:
hmm, in looking at daemon/container_windows.go I do see a DefaultPathEnv value being set so I'm wondering whether it would be better to remove the windows specific code here and change that code in container_windows.go to be just "" instead. Otherwise, what's the point of that assignment?